### PR TITLE
Updates to the styling

### PR DESCRIPTION
**Pull request by King-Tut-Tut**
Bootstrap alerts in markdown.

### CODE REVIEW

File: _articles/getting-started/import-from-1password.md [hunk 1]
```diff
@@ -13,14 +13,11 @@ follow the proper steps below.
...
-This set of instructions is only for:
-- 1Password 4 users on macOS and Windows
-- 1Password 6 users on macOS
-
-These instructions may vary slightly for macOS users.
-```
+<div class="alert alert-info" style="margin-top:18px;">
+    <p>This set of instructions is only for:</p>
+    <ul><li>1Password 4 users on macOS and Windows</li><li>1Password 6 users on macOS</li></ul>
+    <br><p>These instructions may vary slightly for macOS users.</p>
```
Comment (kspearrin) [inline _articles/getting-started/import-from-1password.md]: Shouldn't need the `<br>` here

File: _articles/getting-started/import-from-1password.md [hunk 2]
```diff
@@ -13,14 +13,11 @@ follow the proper steps below.
...
 
-```
-!!!!!! IMPORTANT !!!!!!
-This set of instructions is only for:
-- 1Password 4 users on macOS and Windows
-- 1Password 6 users on macOS
-
-These instructions may vary slightly for macOS users.
-```
+<div class="alert alert-info" style="margin-top:18px;">
```
Comment (kspearrin) [inline _articles/getting-started/import-from-1password.md]: can we create a style for the margin?

File: css/styles.scss [hunk 3]
```diff
@@ -175,6 +175,18 @@ footer {
...
 
+.panel-article .alert {
+    margin-top: 18px;
+}
+
+.panel-article .alert ul, .panel-article .alert ol {
+    padding-left: 20px;
+}
+
+ul + p, ol + p {
```
Comment (kspearrin) [inline css/styles.scss]: we only want this on `.panel-article .alert`

Comment (King-Tut-Tut) [inline css/styles.scss]: Oops

**Pull request merged**